### PR TITLE
Fix asset compilation error

### PR DESCRIPTION
**Pull request by jchapin**
Fixed the asset pipeline errors, manually compiled assets.

screen shot 2016-02-18 at 5 20 05 pm

screen shot 2016-02-18 at 5 19 49 pm

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -44,6 +44,8 @@ class Application < Rails::Application
 
     config.assets.debug = false
 
+    config.assets.initialize_on_precompile = false
```
Comment (jchapin) [inline config/application.rb]: Allowed recompilation of assets.

File: config/environments/production.rb [hunk 2]
```diff
@@ -49,6 +49,7 @@
 
   # Precompile additional assets (application.js, application.css, and all non-JS/CSS are already added)
   # config.assets.precompile += %w( search.js )
+  config.assets.precompile += %w( ie.css )
```
Comment (jchapin) [inline config/environments/production.rb]: Had to specifically include the ie.css file since it wasn't SASS.

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  config.secret_key = '91dc3d9814f6e565eb43badf13f1562fdc9a2987fb599ac60f99159721fc7910f0222d2b6f545a298159aab911e8f9829b07730ffe4011886532e91df0257821'
```
Comment (jchapin) [inline config/initializers/devise.rb]: Devise key, should be an environment var.

**Comment by jchapin** [comment]
@RanSolo @cbradshaw needed to change a few things to get the style compiling correctly.

**Comment by RanSolo** [comment]
:+1: this is the rabbit hole stuff for me.  I can get lost getting stuff compiling right.

**Pull request merged**